frontend/src/components/FilterCategory.tsx

```diff
@@ -0,0 +1,20 @@
+import FilterButton from './FilterButton'
+
+export default function FilterCategory({
+  title,
+  options,
+}: {
+  title: string
+  options: string[]
+}) {
```

Comment:
Optional but good practice would be to define your props outside the component: 

```
interface FilterCategoryProps {
  title: string;
  options: string[];
}

export default function FilterCategory({ title, options }: FilterCategoryProps) {
  // Component logic here
}
```

or for arrow functions:

```
interface FilterButtonProps {
  label: string; 
  onClick?: () => void;  (once we have that)
}

const FilterButton: React.FC<FilterButtonProps> = ({ label, onClick }) => {
```

Could we do this to all the components?